slices/gcc-14-base.yaml

```
@@ -0,0 +1,6 @@
+package: gcc-14-base
+
+slices:
+  copyright:
+    # we do not need to list the copyright file
+    # since chisel will add it anyway.
```

We're now explicitly adding the copyright to all slices such that we can track their provenance.

```suggestion
    /usr/share/doc/gcc-14-base/copyright:
```